.github/workflows/test-automation.yml

```
@@ -37,6 +37,17 @@ jobs:
           NODE_ENV: test
           CI: true
+          METRICS_API_KEY: test-metrics-key
+          ADMIN_API_KEY: test-admin-key
+          SENTRY_DSN: https://test@sentry.io/test
+          VERCEL_URL: http://localhost:3000
+          # Mock environment variables for testing
+          BREVO_API_KEY: mock-test-key
+          STRIPE_PUBLISHABLE_KEY: pk_test_mock
+          STRIPE_SECRET_KEY: sk_test_mock
+          DATABASE_URL: file::memory:?cache=shared
+          ADMIN_API_KEY: test-admin-key
```

actionlint (1.7.7)

key "ADMIN_API_KEY" is duplicated in env. previously defined at line:41,col:11. note that this key is case insensitive

(syntax-check)